crates/next-core/src/next_image/module.rs

```diff
@@ -56,7 +56,9 @@ impl StructuredImageModuleType {
         blur_placeholder_mode: BlurPlaceholderMode,
         module_asset_context: ResolvedVc<ModuleAssetContext>,
     ) -> Result<Vc<Box<dyn Module>>> {
-        let static_asset = StaticUrlJsModule::new(*source).to_resolved().await?;
+        let static_asset = StaticUrlJsModule::new(*source, Some(rcstr!("client")))
```

Comment:
does this need to be a string, could we introduce an enum for assetcontext?

Reply:
Currently, turbopack doesn't have a concept of "server" or "client" layer. That's only in the Next.js side
If it's an enum, then there is no way for the user (i.e. Next.js as the consumer of Turbopack crates) to configure it.